foundry/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM alpine:3.18 as build-environment
```

Comment:
I thought we had migrated from alpine to ubuntu for builds, due to musl vs  glibc problems?

Reply:
We copied foundry's own Dockerfile